templates/master/index.js

```diff
@@ -165,6 +165,56 @@ module.exports={
         "Type":"String",
         "Default":"EMPTY"
     },
+    "ElasticSearchInstanceType": {
+      "Type":"String",
+      "Description":"OpenSearch instance type to use for domain.",
```

Comment:
Let's include a link in the parameter description to the pricing page so that it is easy for users to come to the right decision:

`OpenSearch instance type to use for the domain. Default recommendation for production deployments is m6g.large.search (see https://aws.amazon.com/opensearch-service/pricing for other options)`